src/Bridge/Symfony/Bundle/Resources/views/SwaggerUi/index.html.twig

```diff
@@ -11,7 +11,8 @@
     {% endblock %}
 
     {# json_encode(65) is for JSON_UNESCAPED_SLASHES|JSON_HEX_TAG to avoid JS XSS #}
-    <script id="swagger-data" type="application/json">{{ swagger_data|json_encode(65)|raw }}</script>
+    <script id="swagger-data" type="application/json" data-oauth-redirect-url="{{ absolute_url(asset('bundles/apiplatform/swagger-ui/oauth2-redirect.html')) }}">{{ swagger_data|json_encode(65)|raw }}</script>
```

Comment:
`data-oauth-redirect-url` doesn't belong here. It should be part of `swagger_data`. :smile: